jobs/coin_stats.go

```diff
@@ -108,6 +124,46 @@ func (j *CoinStatsJob) run(ctx context.Context) error {
 	return nil
 }
 
+func (j *CoinStatsJob) updateStats(ctx context.Context, coin ArtistCoin) error {
+	overview, err := j.birdeyeClient.GetTokenOverview(ctx, coin.Mint, "24h")
```

Comment:
if birdeye fails we probably do still want to do the dbc calls right?

Reply:
wait we do

Reply:
don't we return out?

Reply:
duh